### ISSUE TITLE
Unable to join any Skiff user's workspace and vice-versa

### ISSUE THREAD

**Issue by x80486** [skiff-org/skiff-mail#19]
I was testing this feature with a couple of friends. Basically they signed-up with an invitation link that I shared. After that, I went to `Skiff Pages` and created a new team with `invite-only` access; then I tried to "share" it by inviting them with their newly created `Skiff` e-mail address. They receive the invitation e-mail (`You’re invited to a workspace...`) but it states that an account should be created even though they are `Skiff` users already. Eventually the (new) sign-up process fails because the e-mail addresses are taken, so we use the `Log In` option instead...and nothing happens: no one is in the workspace and/or team. They are in `pending` state — which I can see when I go to `Settings > Workspace`.

All the previous happened when I was inviting them as `Editor`s; on the other hand, if I use the `Admin` role, I get this error instead:

![image](https://user-images.githubusercontent.com/34633617/198899122-9c5a250b-12a6-4d53-bdee-fbbd9647f8c6.png)

**Comment by amilich** [comment]
Got it. You should be able to invite any other skiff user (use the full email address) to any folder or document.

To invite them to your full workspace, they will need to sign up with the invite link. Does that not work? They'll need to create a new account for every workspace.

**Comment by x80486** [comment]
Yes, the invitation is done using their e-mail addresses :+1:

I'm inviting them to a newly created team. For instance, there is `Everyone` (which cannot be removed), then I create another one, click on the three-dots and select `Share`...then I add the e-mail address, one by one, and choosing the role accordingly. Not sure if that counts as "inviting them to my full workspace", but the message on the top says: `Share “The Team Name That I Just Created”`.

**Comment by xendarboh** [comment]
Confirming this issue. My team did everything there was to do and could find no way to have a shared workspace. It's broken.

The invite email very unintuitively says "Create an account to join" (when the user already has an account).

> To invite them to your full workspace, they will need to sign up with the invite link. Does that not work?

nope. new fresh empty skiff account, no workspace access. (the invite went to previously existing account.)

> They'll need to create a new account for every workspace.

hard sell.

**Comment by amilich** [comment]
> Confirming this issue. My team could find no way to have a shared workspace. It's broken.
> 
> The invite email very unintuitively says "Create an account to join" (when the user already has an account).
> 
> > To invite them to your full workspace, they will need to sign up with the invite link. Does that not work?
> 
> nope. new fresh empty skiff account, no workspace access. (the invite went to previously existing account.)
> 
> > They'll need to create a new account for every workspace.
> 
> hard sell.

Hello! To share a workspace, you need to send an invite from the organization settings page - https://app.skiff.com/dashboard?settingTab=organization - and then create a new account. There's a limit of just 1 workspace per account - just like Microsoft Outlook, Google Workspace, etc. where a workspace can manage permissions for all subusers (say if you wanted to enforce 2fa).

**Comment by amilich** [comment]
> Yes, the invitation is done using their e-mail addresses 👍
> 
> I'm inviting them to a newly created team. For instance, there is `Everyone` (which cannot be removed), then I create another team (say `The Team Name That I Just Created`). On that same team I then click on the three-dots and select `Share`, I add the e-mail addresses, one by one, and choosing the role accordingly — I've done this one at a time and in bulk. Not sure if that counts as "inviting them to my full workspace", but the message on the top says: `Share “The Team Name That I Just Created”`.

Yes, this should work. However, your teammates will need new accounts if they want to join the existing workspace.
